share/linux/org.keepassxc.KeePassXC.desktop.in

```diff
@@ -37,7 +37,7 @@ Comment[et]=Kogukonna arendatav port Windowsi programmist KeePass Password Safe
 Comment[ru]=Разработанный сообществом порт Windows-приложения KeePass Password Safe
 Exec=keepassxc %f
 TryExec=keepassxc
-Icon=keepassxc
+Icon=@APP_ICON@
```

Comment:
@droidmonkey we could rename that to `APP_ICON_NAME` to be more explicit.

Reply:
Concur

Reply:
This is done.